plugins/module_utils/ndi.py

```diff
@@ -420,7 +420,16 @@ def get_aci_class(self, prefix):
             return "spanSpanLbl"
         elif prefix == "ctx":
             return "fvCtx"
+        elif prefix == "lnodep":
+            return "l3extLNodeP"
+        elif prefix == "rsnodeL3OutAtt":
+            return "l3extRsNodeL3OutAtt"
+        elif prefix == "rt":
+            return "ipRouteP"
+        elif prefix == "nh":
+            return "ipNexthopP"
         else:
+            self.nd.fail_json(msg="Unsupported ACI class identifier prefix: {0}".format(prefix))
             return False
```

Comment:
Can we remove ```return False```  since it is no longer required?

Reply:
Yes you are absolutely right !